.gitignore

```diff
@@ -3,3 +3,4 @@
 /coverage
 docs
 node_modules
+.vscode
```

Comment:
Very minor thing here, but can you add an .editorconfig file in this PR than enforces a LF at end of files and add one here?

Reply:
fixed in https://github.com/troyready/serverless-artifact-cacher/pull/3/commits/557332fa22b21d6a481fd5f9f5b3185f71137357